packages/libs/error-stack/src/fmt/mod.rs

```diff
@@ -12,6 +12,12 @@
 //! Hooks can be attached through the central hooking mechanism which `error-stack`
 //! provides via [`Report::install_debug_hook`].
 //!
+//! Hooks are called for contexts which provide additional values through [`Error::provide`] or
+//! [`Context::provide`] and attachments which are added via [`.attach()`].
```

Comment:
```suggestion
//! [`Context::provide`] and attachments which are added via [`Report::attach()`].
```

Reply:
fixed in [d388a1b](https://github.com/hashintel/hash/pull/974/commits/d388a1b9f48f3acad5d61e3a7afd6dabf371897e)